stormpath-export

```diff
@@ -8,7 +8,6 @@ Easily export your Stormpath (https://stormpath.com/) user data.
 Usage:
   stormpath-export configure
   stormpath-export [(<location> | -l <location> | --location <location>)]
-  stormpath-export [(-b <base_url> | --base-url <base_url>)]
```

Comment:
Heyo! We need to leave this in so someone can supply the base url via the command line if they need to.